free-programming-books-ru.md

```diff
@@ -226,9 +225,9 @@
 
 ### Ruby
 
-* [Ruby за двадцать минут](https://www.ruby-lang.org/ru/documentation/quickstart/)
-* [Ruby](https://github.com/Krugloff/rus_ruby_book) - Круглов А.
 * [Ruby](https://ru.wikibooks.org/wiki/Ruby) - Викиучебник
+* [Ruby](https://github.com/Krugloff/rus_ruby_book) - Круглов А.
```

Comment:
228/229 - If possible, add something like "Ruby Wikibook" and "Ru Ruby book" to differentiate sources